README.md

```diff
@@ -3,10 +3,14 @@
 
 # Setup
 
+Make sure to be connected to the VPN to ensure connectivity to Docmosis dev instance. 
```

Comment:
I think we need to add a note about needed the DOCMOSIS_TEMPLATES_ENDPOINT_AUTH

Reply:
Good point.